assets/i18n/pt-br.json

```diff
@@ -105,6 +105,7 @@
 	"I18N_CLASSROOM_CALLOUT_HEADING_1": "Fundamentos de Matemática",
 	"I18N_CLASSROOM_CALLOUT_HEADING_2": "Apresentando: A Sala de Aula da Oppia",
 	"I18N_CLASSROOM_CALLOUT_PARAGRAPH_1": "Confira o primeiro curso na nova Sala de Aula da Oppia! Planos de aula eficientes, criados por professores, para que você possa dominar importantes disciplinas acadêmicas.",
+	"I18N_CLASSROOM_MATH_TITLE": "Matemática",
```

Comment:
Drop the changes from this file before submitting (otherwise translatewiki won't be happy).